### DISCUSSION TITLE
List dependencies locally

### DISCUSSION THREAD

**Discussion by ghost** [renovatebot/renovate#13526] [Request Help] [answered]
Is there a way to use the CLI to list dependencies found in a repository locally?

I'm trying to debug a regex manager and I would like renovate to list the detected dependencies without having to push the changes for every attempt.

For instance, if I have the following in a Dockerfile:
```
# renovate: datasource=npm depName=yarn
ENV YARN_VERSION=1.19.1
```

is there a way for me to test that renovate detects it locally?

**Comment by viceice** [comment]
nope

**Comment by pulasthibandara** [comment, marked as answer]
Possible option for people coming across this: https://docs.renovatebot.com/modules/platform/local/
